### PR TITLE
Added translation for dutch and lang param in module

### PR DESCRIPTION
**Pull request by amavis442**
Without $lang in module yii can gives an error. Also added translation for dutch.

### CODE REVIEW

File: messages/nl/core.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Language 'EN'.
+ * 
+ * @author Vasilij Belosludcev http://mihaly4.ru
+ * @version 1.0.0
```
Comment (bupy7) [inline messages/nl/core.php]: new version will 1.1.1 =)

File: messages/nl/core.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Language 'EN'.
+ * 
+ * @author Vasilij Belosludcev http://mihaly4.ru
```
Comment (bupy7) [inline messages/nl/core.php]: You author, not me.

File: messages/nl/core.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Language 'EN'.
```
Comment (bupy7) [inline messages/nl/core.php]: Language is 'NL'

File: Module.php [hunk 4]
```diff
@@ -76,6 +76,8 @@ class Module extends \yii\base\Module
      */
     public $urlToFiles;
     
+    public $lang;
```
Comment (bupy7) [inline Module.php]: where this uses? i not understand.

**Comment by amavis442** [comment]
Corrected copyright notice and version number. Reverted $lang in module. I used $lang in an override controller for ManageController so i could change the view _form with an extra field for an menu item. My apologies for this inconvenience.

**Comment by bupy7** [comment]
Thanks!

**Pull request merged**